ls1intum/ArTEMiS
# Complaints for modeling and text exercises

### Checklist

### Description
In https://github.com/ls1intum/ArTEMiS/pull/353 we extended the existing complaints mechanism and integrated it into modeling exercises. In this PR we re-enable the complaints for text exercises and integrate the new changes. See https://github.com/ls1intum/ArTEMiS/pull/353 for further details.

### Steps for Testing
1. Log in to ArTEMiS as a student
2. Navigate to assessed text submission
3. Click on "Complain"
4. Enter and submit a complaint
5. Log in to ArTEMiS as a tutor (not the assessor of the submission from above)
6. Navigate to tutor exercise dashboard for the corresponding exercise
7. See list of complaints and click on "Evaluate the complaint"
8. Reply to complaint
9. Log in as student again
10. Check response for previously submitted complaint
